README.md

```diff
@@ -113,16 +123,23 @@ data_MP = MODData.load('path/to/moddata/MP_2018.6')
 
 ```
 
-The latter MODData (MP_2018.6) is very usefull for predicting a learned property on all structures from the Materials Project:
+The MP MODData on [figshare](https://figshare.com/articles/dataset/Materials_Project_MP_2018_6_MODData/12834275)
+(MP_2018.6) is very usefull for predicting a learned property on all structures from the Materials Project:
```

Comment:
Do we release the MP model too? The example line then shows model predictions...

```suggestion
The MP MODData on [figshare](https://figshare.com/articles/dataset/Materials_Project_MP_2018_6_MODData/12834275) (MP_2018.6) is very useful for predicting a learned property on all structures from the Materials Project:
```